vendors/nxp/boards/lpc54018iotmodule/aws_demos/config_files/aws_demo_config.h

```diff
@@ -42,17 +42,17 @@
 #define CONFIG_MQTT_DEMO_ENABLED
 
 /* Default configuration for all demos. Individual demos can override these below */
-#define democonfigDEMO_STACKSIZE                          ( configMINIMAL_STACK_SIZE * 15 )
+#define democonfigDEMO_STACKSIZE                          ( configMINIMAL_STACK_SIZE * 20 )
 #define democonfigDEMO_PRIORITY                           ( tskIDLE_PRIORITY + 1 )
 #define democonfigNETWORK_TYPES                           ( AWSIOT_NETWORK_TYPE_WIFI )
 
 #define democonfigSHADOW_DEMO_NUM_TASKS                   ( 2 )
-#define democonfigSHADOW_DEMO_TASK_STACK_SIZE             ( configMINIMAL_STACK_SIZE * 4 )
+#define democonfigSHADOW_DEMO_TASK_STACK_SIZE             ( configMINIMAL_STACK_SIZE * 15 )
 #define democonfigSHADOW_DEMO_TASK_PRIORITY               ( tskIDLE_PRIORITY + 1 )
-#define shadowDemoUPDATE_TASK_STACK_SIZE                  ( configMINIMAL_STACK_SIZE * 5 )
+#define shadowDemoUPDATE_TASK_STACK_SIZE                  ( configMINIMAL_STACK_SIZE * 10 )
 
 #define democonfigMQTT_ECHO_TLS_NEGOTIATION_TIMEOUT       pdMS_TO_TICKS( 12000 )
-#define democonfigMQTT_ECHO_TASK_STACK_SIZE               ( configMINIMAL_STACK_SIZE * 11 )
```

Comment:
There is change that Abhijit done to change MQTT stack size to default size. https://github.com/aws/amazon-freertos/pull/1032. Can you please help check if that change applies here?

Reply:
I see that you already updated the PR to remove the stack size similar to the above mentioned changes. Thanks